caddyhttp/httpserver/context.go

```diff
@@ -57,6 +58,24 @@ func (c Context) Header(name string) string {
 	return c.Req.Header.Get(name)
 }
 
+// Env gets a map of the environment variables.
+func (c Context) Env() map[string]string {
+	os_env := os.Environ()
```

Comment:
Will update shortly. Noted for next time ;)

On 14 Aug 2016 22:25, "Thomas Boerger" <notifications@github.com> wrote:

> In caddyhttp/httpserver/context.go
> <https://github.com/mholt/caddy/pull/1035#discussion_r74708717>:
>
> > @@ -57,6 +58,24 @@ func (c Context) Header(name string) string {
> >  	return c.Req.Header.Get(name)
> >  }
> >
> > +// Env gets a map of the environment variables.
> > +func (c Context) Env() map[string]string {
> > +	os_env := os.Environ()
>
> Convention for variable names is camel-case
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mholt/caddy/pull/1035/files/2e5872d948564c131164b487a22682e346452d8b#r74708717>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA3VRPL5f9PrfRC85YQYPSL72rBPeO4hks5qf4evgaJpZM4Jj-Yx>
> .
>